composer.json

```diff
@@ -38,7 +38,7 @@
         "sonata-project/easy-extends-bundle": "^2.1",
         "sonata-project/doctrine-extensions": "^1.0",
         "sonata-project/classification-bundle": "^3.0",
-        "knplabs/gaufrette": "^0.1.6",
+        "knplabs/gaufrette": "~0.2",
```

Comment:
If we should keep `^0.1.6` we could close this PR, because `^0.1.6 | ^0.2` is the same as `^0.1.6`.